src/aero_mode.F90

```diff
@@ -557,7 +557,7 @@ subroutine aero_mode_sample_radius(aero_mode, aero_data, aero_weight, &
     type(aero_weight_t), intent(in) :: aero_weight
     !> Sampled radius (m).
     real(kind=dp), intent(out) :: radius
-    !>
+    !> Control factor to limit out of range sampling of normal distribution.
```

Comment:
@mwest1066 please make sure this comment is clear.

Reply:
I'm actually a little bit unclear on exactly how our threshold is working for normal distribution sampling. We also appear to use `size_factor` in `aero_mode.F90` but `threshold` in `rand.F90` to refer to the same thing?

I traced out the key equations below, as far as I can follow them. Could you write these up in latex and explain why we are using these equations? If it's not clear to you then I'm happy to meet to discuss it.

List of key equations that I could find:

In `scenario.F90`:
```
sample_timescale_effective = max(1.0, min(sample_timescale, &
          env_state%elapsed_time))
     characteristic_factor = sample_timescale_effective / delta_t
```

In `aero_state.F90`:
```
size_factor = min((1.0d0 / (characteristic_factor * n_samp_avg)), &
               0.32d0)
```

In `rand.F90`:
```
if (1.0d0 - abs(erf(z0/(2.0**.5))) >= threshold) acceptable = .true.
```

Reply:
We discussed this and updated the notation and variable names. We'll document this in the upcoming paper.